TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/BasicAuto.java

```diff
@@ -0,0 +1,56 @@
+package org.firstinspires.ftc.teamcode.opmodes;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+
+import org.firstinspires.ftc.teamcode.hardware.Drivebase;
+import org.firstinspires.ftc.teamcode.hardware.VisionCamera;
+import org.firstinspires.ftc.teamcode.utility.FieldSide;
+
+import java.util.Objects;
+
+@Autonomous(name = "BasicAuto")
+public class BasicAuto extends LinearOpMode {
+    @Override
+    public void runOpMode() {
+        // ALWAYS SET TELEMETRY AUTOCLEAR to TRUE!!!
+        // INIT CODE.
+        telemetry.setAutoClear(true);
+
+        telemetry.addData("Status", "Initializing");
+        telemetry.update();
+
+        Drivebase drivebase = new Drivebase(hardwareMap, this::opModeIsActive);
+        VisionCamera camera = new VisionCamera(hardwareMap, FieldSide.BlueClose);
+
+        telemetry.addData("Status", "Initialized");
+        telemetry.update();
+        waitForStart();
+        // RUNNING CODE.
+
```

Comment:
You should add a logic check to ensure that the op mode is active.

Reply:
I think we don't need to do that, because waitForStart ensures our program is in the right state.